all-builds.nix

```diff
@@ -85,6 +95,14 @@ let
           ghcjs
           serverSkeletonExe
           ;
+      } // lib.optionalAttrs (system == "x86_64-linux") {
+        server =
+          if profiling
+          then {} # build with profiling has deps marked as broken (e.g. ‘th-orphans-0.13.7’)
+          else
+            # collect expands these into the entirety of nixpkgs, which has broken pkgs
+            let avoidBuildingNixpkgs = attrs: builtins.removeAttrs attrs [ "config" "options" "pkgs" ];
```

Comment:
This blacklisting feels odd to me, maybe `collect` is doing more harm than good?